### DISCUSSION TITLE
what the f%%% is the new home page? really. do we need another social network when we are trying to focus on getting stuf done?

### DISCUSSION THREAD

**Discussion by sergey-imprezz** [community/community#101935] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

subj

**Comment by stuiterveer** [comment]
![image](https://github.com/community/community/assets/8035044/1c9a65c0-e84f-44fa-a7fd-80963e53c7eb)
"you can customize your feed exactly how you like it": with clutter, or less clutter. But we're still giving you clutter.

Way to f-ing go Microsoft. Your predatory selections in Windows made its way to Github.
